src/App.vue

```diff
@@ -41,48 +40,19 @@ import {} from './assets/global.scss'
 import ModalCreateOrUpdate from './components/Common/Environments/ModalCreateOrUpdate'
 import ModalDelete from './components/Common/Environments/ModalDelete'
 import ModalImport from './components/Common/Environments/ModalImport'
-import MainSpinner from './components/Common/MainSpinner'
 
 export default {
   name: 'KuzzleAdminConsole',
   components: {
     ModalCreateOrUpdate,
     ModalDelete,
-    ModalImport,
-    MainSpinner
+    ModalImport
   },
   data() {
     return {
-      initializing: true,
       environmentId: null
     }
   },
-
-  async mounted() {
```

Comment:
* This must be restored
* The route guard must be deleted
* we need to add to this the redirection to `CreateEnvironment` in case no environments are present after the loading